WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -49,7 +48,7 @@ private extension BlazeCampaignDashboardViewHostingController {
     private var productSelectorViewController: ProductSelectorViewController {
         let productSelectorViewModel = ProductSelectorViewModel(
             siteID: viewModel.siteID,
-            purchasableItemsOnly: true,
+            purchasableItemsOnly: false,
```

Comment:
Can we skip sending a value for `purchasableItemsOnly` as `false` is the default value?

Reply:
Done in the Controller, it no longer sends the value.